packages/core/contracts/data-verification-mechanism/implementation/DesignatedVotingV2Factory.sol

```diff
@@ -2,16 +2,15 @@
 pragma solidity 0.8.16;
 
 import "./DesignatedVotingV2.sol";
+import "../../common/implementation/MultiCaller.sol";
 
 /**
  * @title Factory to deploy new instances of DesignatedVotingV2 and look up previously deployed instances.
  * @dev Allows off-chain infrastructure to look up a hot wallet's deployed DesignatedVoting contract.
  */
-contract DesignatedVotingV2Factory {
```

Comment:
the changes made to this contract enable two main things:
1) we can now multicall
2) we can now deploy a designated voting contract _while_ setting the voter address

additionally, this PR removes the `designatedVotingContracts` mapping which was redundant.

Reply:
Contract seems way simpler +1